aspnetcore/mvc/controllers/filters.md

```diff
@@ -367,6 +372,38 @@ For an `IAsyncResultFilter` a call to `await next` on the `ResultExecutionDelega
 
 The framework provides an abstract `ResultFilterAttribute` that you can subclass. The [AddHeaderAttribute](#add-header-attribute) class shown earlier is an example of a result filter attribute.
 
+### IAlwaysRunResultFilter and IAsyncAlwaysRunResultFilter
+
+The <xref:Microsoft.AspNetCore.Mvc.Filters.IAlwaysRunResultFilter> and <xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncAlwaysRunResultFilter> interfaces declare an <xref:Microsoft.AspNetCore.Mvc.Filters.IResultFilter> implementation that runs for all action results, even if the response is short-circuited in the request processing pipeline.
```

Comment:
Here you say always run, even when short circuited, but then below you say except. Can you add the exception cases here? No pun intended.